schema/crdb/dbinit.sql

```diff
@@ -312,6 +326,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.physical_disk (
     -- FK into the Sled table
     sled_id UUID NOT NULL,
 
+    -- Describes how the control plane manages this disk
+    state omicron.public.physical_disk_state NOT NULL,
+
```

Comment:
TODO(me): add an index to look up all "draining" disks". 

Also maybe update the terminology here to match https://github.com/oxidecomputer/omicron/issues/4719

Reply:
Nit: I should use the term "deactivating" here, rather than "draining", because "draining" can refer to cooperative removal (e.g. in the context of draining instances from a sled)